src/tally.F90

```diff
@@ -2478,7 +2498,7 @@ subroutine score_surface_current(p)
 
               ! Outward current on d1 min surface
               if (all(ijk0 >= 1) .and. all(ijk0 <= m % dimension)) then
```

Comment:
This line looks to be problematic because `ijk0` is always size 3 but `m % dimension` could be less. When I try to run the 1D and 2D mesh filter tests, they both fail on this line.